docs/source/common/loggers.rst

```diff
@@ -28,6 +28,56 @@ the path.
         img = ...
         log_image(img, self.trainer.log_dir)
 
+Weights and Biases
+==================
+
+`Weights and Biases <https://docs.wandb.ai/integrations/lightning/>`_ is a third-party logger.
+To use :class:`~pytorch_lightning.loggers.WandbLogger` as your logger do the following.
+First, install the package:
+
+.. code-block:: bash
+
+    pip install wandb
+
+Then configure the logger and pass it to the :class:`~pytorch_lightning.trainer.trainer.Trainer`:
+
+.. code-block:: python
+
+    from pytorch_lightning.loggers import WandbLogger
+
+    # instrument experiment with W&B
+    wandb_logger = WandbLogger(project="MNIST", log_model="all")
+    trainer = Trainer(logger=wandb_logger)
+
+    # log gradients and model topology
+    wandb_logger.watch(model)
+
+The :class:`~pytorch_lightning.loggers.WandbLogger` is available anywhere except ``__init__`` in your
+:class:`~pytorch_lightning.core.lightning.LightningModule`.
+
+.. code-block:: python
+
+    class MyModule(LightningModule):
+        def any_lightning_module_function_or_hook(self):
+            some_img = fake_image()
+            # Option 1
+            self.logger.experiment.log({"generated_images": [wandb.Image(some_img, caption="...")]})
+            # Option 2 for specifically logging images
+            self.logger.log_image(key="generated_images", images=[some_img])
+
+To visualize wandb in a jupyter notebook environment, run the following command in a jupyter cell:
+
+.. code-block:: bash
+
+    %%wandb
```

Comment:
as @krshrimali mentioned, the addition of this part is fine. Other seems unimportant. You can extend this one too if there's anything more to add here.